scania-digital-design-system/tegel
# build(stencil.config): updated stencil config

**How to test**  
1. Check out branch
2. CD into core
3. Run npm i
4. run npm build
5. Check the genereated components folder (next to dist)
6. Check the index.d.ts and check that there are no issues
